src/framework/global/io/path.cpp

```diff
@@ -246,7 +246,9 @@ std::string mu::io::pathsToString(const paths& ps, const std::string& delim)
     }
 
     for (size_t i = 0; i < delim.length(); ++i) {
-        result.pop_back();
+        if (result.size()) {
```

Comment:
It is better to rewrite this code as follows:

```
for (size_t i = 0; i < delim.length(); ++i) {
    if (result.empty()) {
        break;
    }

    result.pop_back();
}
```

Reply:
Sure, I had considered the same but realized that after the commit. 
I'll push it right away!

Reply:
Or even like this:

result.erase(result.length() - delim.length())